bundles/org.openhab.ui/web/src/components/item/metadata/item-metadata-menu.vue

```diff
@@ -4,10 +4,22 @@
       <f7-list>
         <ul>
           <f7-list-item
-            v-for="namespace in metadataNamespaces.filter((n) => item.metadata[n.name])" :key="namespace.name"
-            :link="'/settings/items/' + item.name + '/metadata/' + namespace.name"
-            :title="namespace.label"
-            :after="(item.metadata[namespace.name]) ? item.metadata[namespace.name].value : 'Not Set'" />
+            v-for="namespace in standardNamespaces" :key="namespace"
+            :link="'/settings/items/' + item.name + '/metadata/' + item.metadata[namespace].id"
+            :title="item.metadata[namespace].label"
+            :after="(item.metadata[namespace].id) ? item.metadata[namespace].id : 'Not Set'" />
+        </ul>
+      </f7-list>
+    </f7-card-content>
+    <hr>
+    <f7-card-content v-if="item.metadata && Object.keys(item.metadata).filter((n) => n !== 'semantics').length > 0">
+      <f7-list>
```

Comment:
That `<hr>` is ugly, use this:

```suggestion
        </ul>
        <ul v-if="customNamespaces && customNamespaces.length > 0">
          <f7-list-item divider title="Custom" />
          <f7-list-item
```
Filtering out the semantics namespace can happen in the computed property.

Reply:
Adding this instead of hr looks like 
<img width="788" alt="image" src="https://user-images.githubusercontent.com/5937600/213794175-f068f28b-dfcb-4a53-aced-be673f3b9b4b.png">
but I think the "custom" is not looking good.

I would recommend to leave the title "Custom" out:

<img width="759" alt="image" src="https://user-images.githubusercontent.com/5937600/213794360-7dbb5265-08d7-4864-bc0f-a476a1530192.png">

What about "semantics"? Should be add it to the known "MetadataNamespaces" or does it belong to "custom" or should we filter it out completely (not show it all)?

Reply:
> I would recommend to leave the title "Custom" out

Okay, it looks slightly better indeed.

> What about "semantics"? Should be add it to the known "MetadataNamespaces" or does it belong to "custom" or should we filter it out completely (not show it all)?

This namespace is provided by a different MetadataProvider so it's not editable (in fact you already have the info in "Semantic Classification" on the item page). So it shouldn't be displayed at all in this list.
By the way, this PR allows the listing of every namespace (except "semantics") as they were editable, as there's no way to determine from which MetadataProvider they come. The problem is, if they're from a non-managed MetadataProvider (for example provided by an add-on) they will be presented as editable but saving will have adverse effects (they would then be defined twice on different providers I think, so I don't know what will happen). See https://github.com/openhab/openhab-core/issues/3281 for more info.